Jellyfin.Database/Jellyfin.Database.Implementations/Jellyfin.Database.Implementations.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net9.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
I don't think we use ImplicitUsings anywhere?

Reply:
its the default for new projects
In or Out?